Resources/Prototypes/Catalog/VendingMachines/Inventories/theater.yml

```diff
@@ -30,6 +30,9 @@
     ClothingHeadHatPumpkin: 2
     ClothingHeadHatShrineMaidenWig: 2
     ClothingOuterSuitShrineMaiden: 2
+    ClothingHeadHatWitch1: 2
```

Comment:
Remind me later that we pretty desperately need a way to get these into player's hands that don't involve the theater vendor. Awhile ago I declared a requirement that, "No clothing vendor is allowed to have >=100 items in it", for reasons related to the stupid Cargo Arbitration, but also generally because having the clothing restock crate cost more than a crate full of laser rifles is kind of unacceptable. 

Somehow our cargo arbitration for a clothing restock is back up to >3000. 

We can probably get away with reducing the total number of every item in this vendor down to 1.